### DISCUSSION TITLE
Fetch TestFlight feedback by crashs or screenshot

### DISCUSSION THREAD

**Discussion by jc-paris** [fastlane/fastlane#18701] [Ideas]
Hello,

I've been playing with `get_beta_feedback` from #15793. 

I noticed that the App Store Connect private API uses the param `&exists[crash]=true` to filter between crashes and screenshots.

Would it be possible to include this possibility in the `get_beta_feedback` ?

_Or maybe this is already possible without a fasltane update but I didn't find a way to specify this param to fastlane client..._

**Comment by jc-paris** [comment]
Ping @max-ott, as you created the original PR merged in fastlane, you may have some idea on how to do that 😃

**Reply by max-ott** [reply]
I'm not using the endpoint, so I currently have no interest to add this in my free time

**Reply by jc-paris** [reply]
I understand 🙂 
I was wondering if I miss something and this could already be possible. Seems I was right and filtering crash from screenshot is going to require some work.
Thanks for replying !
